test/ui/test_stitch_axis.py

```diff
@@ -0,0 +1,84 @@
+import os
+import pytest
+from RefRed.main import MainGui
+from qtpy import QtCore, QtWidgets
+
+
+wait = 100
+
+
+def test_stitch_plot(qtbot, data_server):
+    """Test the axis adjustment in stitching plot"""
+    # Create the main window.
+    main = MainGui()
```

Comment:
Although this is not C/C++, I would not use `main` as a variable name

Reply:
var name updated as suggested